src/common/utils/gpo.h

```diff
@@ -260,6 +261,12 @@ namespace powertoys_gpo {
     {
         return getConfiguredValue(POLICY_CONFIGURE_ENABLED_REGISTRY_PREVIEW);
     }
+
+    inline gpo_rule_configured_t getDisablePerUserInstallationValue()
+    {
+        return getConfiguredValue(POLICY_DISABLE_PER_USER_INSTALLATION);
```

Comment:
Should we add a mew method that only reads HKLM? In case someone sets the value manually in registry.

Reply:
Well, in that case, user would disable it for himself, right? So,no?

Reply:
I was a bit unclear. What I mean is we have a Machine (HKLM) only policy and checking both. This means that you still manually can set it in HKCU. Not sure if this something we should prevent. 🤔

Reply:
> This means that you still manually can set it in HKCU
Yes, but who can? Current user? If the current user wants to disable per-user install for himself, so be it :)

Reply:
Okay. Then let's not overcomplicating thing and read both. I only want to mention it.